tox.ini

```
@@ -12,7 +12,7 @@ extras =
     tests
 commands =
     {envpython} selftest.py
-    {envpython} -m pytest -W always {posargs}
+    {envpython} -m pytest -vv -W always {posargs}
```

We're not using tox in CI, so we can leave it to the user to add the options if they want to re-run with more verbosity (which would be something like `tox -e py313 -- -vv`):

```suggestion
    {envpython} -m pytest -W always {posargs}
```